utils/gen_onnx_mlir.py

```diff
@@ -421,7 +421,22 @@
       } else {
         resultTypes.push_back(mlir::UnrankedTensorType::get(
           FloatType::getF32(getContext())));
-      }'''
+      }''',
+  "RandomNormal":
+  '''if (auto attr = dtypeAttr()) {
+      if (dtype() == 0) {
+        resultTypes.push_back(mlir::UnrankedTensorType::get(
+            FloatType::getF16(getContext())));
+      } else if (dtype() == 1) {
+        resultTypes.push_back(mlir::UnrankedTensorType::get(
+            FloatType::getF32(getContext())));
+      } else if (dtype() == 2) {
+        resultTypes.push_back(mlir::UnrankedTensorType::get(
+            FloatType::getF64(getContext())));
+      }
```

Comment:
I suggest you fail if dtype > 2
```
  else llvm_unreachable(...);
```
otherwise

Reply:
Good point!